test/index.js

```diff
@@ -111,15 +111,19 @@ test('exhaustive vs non-exhaustive expansion', function (t) {
 
 test('raw curly braces for custom slot types', function (t) {
   var dictionary = {};
-  var slots = {"Artist": "CUSTOM_TYPE"};
-  var template = "{my|your} {favorite|least favorite} fruit is {-|Fruit}";
+  var slots = {"Artist": "CUSTOM_TYPE","ROOM_NAME": "AMAZON.Room"};
+  var template = "{my|your} {favorite|least favorite} fruit is {-|Fruit} {in +ROOM_NAME+|}";
 
   var result = utterances(template, slots, dictionary);
   t.deepEqual(result, [
-    "my favorite fruit is {Fruit}",
-    "your favorite fruit is {Fruit}",
-    "my least favorite fruit is {Fruit}",
-    "your least favorite fruit is {Fruit}"
+    "my favorite fruit is {Fruit} in {ROOM_NAME}",
+    "your favorite fruit is {Fruit} in {ROOM_NAME}",
+	"my least favorite fruit is {Fruit} in {ROOM_NAME}",
+    "your least favorite fruit is {Fruit} in {ROOM_NAME}",
+	"my favorite fruit is {Fruit} ",
```

Comment:
@harrisonhjones If I trim the trailing space it seems to break the "optional terms" test which also has trailing spaces in the result.  I can add a trim to the returned utterances (ie: return utterances.time(); but not sure if that would be desired or not.